Source/rclUE/Public/ROS2Publisher.h

```diff
@@ -22,19 +23,20 @@ class RCLUE_API UROS2Publisher : public UActorComponent
     UROS2Publisher();
 
 public:
-    UFUNCTION()
     virtual void InitializeWithROS2(AROS2Node* InROS2Node)
     {
-        InROS2Node->AddPublisher(this);
+        checkNoEntry();

```

Comment:
Does using checkNoEntry() better than make this function as pure virtual function?

Reply:
Ah right, actually this could invoke `RegisterToROS2Node(InROS2Node);`, so would not have to be called in child class methods.